cobbler/modules/managers/bind.py

```diff
@@ -24,51 +24,39 @@
 import re
 import socket
 import time
+from builtins import object
+from builtins import range
+from builtins import str
 
-from cobbler import clogger
-from cobbler import templar
-from cobbler import utils
+import cobbler.clogger as clogger
+import cobbler.templar as templar
+import cobbler.utils as utils
```

Comment:
I belive that was my fault.

```suggestion
from cobbler import clogger
from cobbler import templar
from cobbler import utils
```